docs/framework/wpf/getting-started/community-feedback.md

```diff
@@ -1,192 +1,124 @@
-  
-### MSDN Community Content Moderation  
- The MSDN Community Content is moderated to ensure that the content is appropriate. As described by the MSDN [Content Moderation Guidelines](http://go.microsoft.com/fwlink/?LinkId=98864), the following types of content are considered inappropriate:  
-  
--   Content that violates the MSDN Community [Code of Conduct](http://go.microsoft.com/fwlink/?LinkId=98865)  
-  
--   Irrelevant, incomprehensible, and test content  
-  
--   Duplicate content  
-  
--   Spam  
-  
--   Documentation feedback  
-  
--   Bug reports  
-  
--   Questions  
-  
- Moderators will process these types of content according to the MSDN Community Content Moderation Guidelines.  
-  
-### MSDN Community Content Language Support  
- MSDN Community Content is available on both the US English and Brazilian Portuguese versions of MSDN Online. Future releases are being considered for Japanese, French, German, Italian, Spanish, Korean, and both Simplified and Traditional Chinese.  
-  
-### MSDN Community Content Frequently Asked Questions  
- Further information can be found in the MSDN Community Content [FAQ](http://go.microsoft.com/fwlink/?LinkId=98866).
+> Don't use the feedback section located at the bottom of each page to send product feedback. These links are for documentation feedback only.
+
+## Forums
+
+The [WPF forum](https://social.msdn.microsoft.com/Forums/vstudio/en-US/home?forum=wpf) is the primary community resource for discussing and resolving issues. Forums facilitate discussion and problem resolution by offering a comprehensive set of supporting features that include:
+
+- Searching.
+- Discussion tracking.
+- Rich formatting for text and code.
+- Visual Studio integration.
+- Most Valued Professional (MVP) and community involvement.
+- Monitoring to ensure posts are responded to in the quickest possible time.
+
+Another option for you to ask questions to the community about WPF is [Stack Overflow](https://stackoverflow.com/questions/tagged/wpf).
+
+### Forum best practices
+
+Using the following best practices help to address issues posted to the WPF forum in the quickest possible time. These practices are applicable to all forums.
+
+#### Search Existing Posts
+
+Some issues occur widely enough that others have faced them before you. Consequently, you can solve your problem quickly, or you can add your input to an existing discussion.
+
+#### Use meaningful titles
+
+Concise, meaningful titles improve the discoverability of your posts. They also make it easier for other WPF forum community members to determine if they can solve your problem.
+
+#### Include appropriate content
+
+Describe the issue and how you’ve tried to work through it. If possible, include supporting code snippets, or the simplest possible sample that demonstrates your issue. All these details help to increase the chance your question will be answered quickly.
+
+## Visual Studio Developer Community
```

Comment:
nit: Should this be sentence case?

Reply:
That's the actual name of the site so I think it's ok like this. But I found another heading that should be sentence case and it's not.